postgis/lwgeom_btree.c

```diff
@@ -131,10 +132,64 @@ PG_FUNCTION_INFO_V1(lwgeom_hash);
 Datum lwgeom_hash(PG_FUNCTION_ARGS)
 {
 	GSERIALIZED *g1 = PG_GETARG_GSERIALIZED_P(0);
+
 	int32_t hval = gserialized_hash(g1);
 	PG_FREE_IF_COPY(g1, 0);
 	PG_RETURN_INT32(hval);
 }
 
+static int
+lwgeom_cmp_abbrev(Datum x, Datum y, SortSupport ssup)
+{
+	/* Empty is a special case */
+	if (x == 0 || y == 0 || x == y)
```

Comment:
When comparing boxes you are doing `/* Empty < Non-empty */` and `/* Non-empty > Empty */`, but here you are doing `NULL = ANYTHING` and `ANYTHING = NULL`. Is that correct?

Reply:
In abbreviated comparator "0" means "I don't know, ask the big guy".

Reply:
:ok_hand: Thanks